RFC-010-core-strategy/RFC-010-core-strategy.md

```diff
@@ -0,0 +1,131 @@
+The information is contained in the Core Strategy document (core-strategy.md).
+
+The Core Strategy document seeks to outline the fundamentals of the project, answering the following questions:
+
+* What is The Good Docs Project?
+* What do we do?
+* Who is our audience?
+* What content do we offer?
+
+This can then drive our content strategy, and work will follow in future RFCs to answer deeper questions about content substance, workflow, structure and governance.
+
+
+## Motivation
+
+There are several benefits to articulating where we are, how we can best support our users and community members, and where we’d like to grow in future.
+
+Documenting our core strategy means that:
+
+**As a project**, we can: 
+
+* Have a touchstone or “North Star” to guide decisions about content produced by the project, as well as communications about the project.
+* Reinforce the foundation that is already underlying our current initiatives.
+* Adapt easily to support our audience. As we grow, our audience will grow and their needs might change. 
+
+**As a community**, we can:
+
+* Hold a shared idea of what we’re about—so that we’re on the same page. 
+* Present a unified idea, and use consistent language, when talking about the identity of our project to others.
+
+**As content producers**, we can: 
+
+* Address content challenges, prioritize them accordingly and form solutions. 
+* Tie each piece of content that we create back to a community or project goal. 
+
+
+## Proposal
+
+That we accept the following related file:
+
+* core-strategy.md
+
+
+## Consequences
+
+An agreed core strategy will help all members of our community, and end users, understand what the project is doing. This will impact the words 
+on our website, how we meet the needs of our users and how we document our own project.
+
+The project will evolve over time, and the core strategy statement in this RFC is not intended to be future-proof. We should revisit the core 
+strategy and vision statement at regular intervals—this might be annually, or if we happen to have a major shift in focus or experience unexpected growth.
+
+
+## Links and prior art
+
+{This section is optional if you want to [link](https://example.com) to other resources.}
+
+
+## Open questions
+
+{This section is optional and is where you can list questions that won't be resolved by this RFC, including those raised in comments from community members.}
+
+
+## Decisions deferred
+
+What’s not covered is the deeper dive into the various pillars of content strategy such as content substance, workflow, structure and governance. 
+These pillars answer questions like:
+
+* What are our content types?
+* Who produces and maintains our content?
+* On what channels and platforms does the content live?
+* How do our users access the content?
+* When and how is content reviewed, evaluated and archived?
+
+This work is intended to follow as future deliverables from the Content Strategy working group.

```

Comment:
Any initial thoughts about how those would be delivered? Amending the core strategy doc?

Do you have a projected time table or "we're going to get to X first, Y second" ranking of issues?

Reply:
Revisiting this at the outset of each release seems like one natural touchpoint, as it's a chance to evaluate where the project wants to go, even if it's the same point on the horizon as it was the last release.

Reply:
@ryanmacklin They won't go into the Core Strategy doc—they'll be documents in their own right. I guess they'll go into the Governance repo, and then our project docs and website. I will add a note about this to the section.

@acpkendo Yes. I thought I had written that somewhere but maybe it was just in my head. I will try to capture that point in a place that makes sense.